addon-api/content-script/Tab.js

```diff
@@ -555,6 +555,11 @@ export default class Tab extends Listenable {
         },
         until: () => [],
       },
+      beforeEditorProfile: {
+        element: () => q("[class^='menu-bar_account-info-group']"),
```

Comment:
This element only exists when signed in, so the editor fullscreen addon won't work when signed out.